packages/Autodiscovery/tests/Source/config/packages/framework.yaml

```diff
@@ -1,8 +1,4 @@
 framework:
-    templating:
-        # needed to enable "twig.loader.filesystem" service
-        engines: ["twig"]
-
```

Comment:
See https://symfony.com/index.php/blog/new-in-symfony-4-3-deprecated-the-templating-component-integration

Reply:
Also related: https://stackoverflow.com/questions/58380812/symfony-4-3-user-deprecated-the-symfony-bundle-twigbundle-loader-filesystemloa